Main.py

```diff
@@ -0,0 +1,3 @@
+Username= input("Kullanıcı Adı Giriniz: ")
```

Comment:
first of all we should create a user class and create a method for checking user email and password